documentation/MeetingMinutes/README.MD

```diff
@@ -1,3 +1,3 @@
 ## RELEVANT INFORMATION
 
-Since 4th April WG Meeting, meeting minutes are placed in [CAMARA Blockchain Public Address WG Wiki Confluence site](https://wiki.camaraproject.org/display/CAM/BlockchainPublicAddress+Meetings)
+Since 4th April WG Meeting, meeting minutes are placed in [CAMARA Blockchain Public Address WG Wiki Confluence site](https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14553270/Minutes+-+BlockchainPublicAddress+Meetings)
```

Comment:
```suggestion
Since 4th April WG Meeting, meeting minutes are placed in [CAMARA Blockchain Public Address WG Wiki Confluence site](https://lf-camaraproject.atlassian.net/wiki/x/thDe)
```